File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    // Prompt the uesr for height until a valid number: 1-8
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // For loop to print each line
    for (int i = 0; i < n; i++)
    {
        // Print spaces
        for (int a = 0; a < n - i - 1; a++)
        {
            printf(" ");
        }

        // Print left hashes
        for (int b = 0; b < i + 1; b++)
        {
            printf("#");
        }

        // Print gap
        printf("  ");

        // Print right hashes
        for (int c = 0; c < i + 1; c++)
        {
            printf("#");
        }

        // Break line
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

# Prompt the uesr for height until a valid number: 1-8
n = 0

while n < 1 or n > 8:
    n = get_int("Height: ")

# For loop to print each line
for i in range(n):
    # Print spaces
    for a in range(n - i - 1):
        print(" ", end="")

    # Print left hashes
    for b in range(i + 1):
        print("#", end="")

    # Print gap
    print("  ", end="")

    # Print right hashes
    for c in range(i + 1):
        print("#", end="")

    # Break line
    print()
